src/fs/file.rs

```diff
@@ -514,6 +514,58 @@ impl<'dir> File<'dir> {
         }
     }
 
+    /// Recursive folder size
+    #[cfg(unix)]
+    pub fn recursive_size(&self) -> f::Size {
+        use crate::fs::RECURSIVE_SIZE_HASHMAP;
+        if self.is_directory() {
+            let Ok(dir) = Dir::read_dir(self.path.clone()) else {
+                return f::Size::None;
+            };
+            let files = dir.files(super::DotFilter::Dotfiles, None, false, false);
+
+            let mut recursive_size: u64 = 0;
+            for fileresult in files {
+                let Ok(file) = fileresult else { continue };
```

Comment:
This feels awkward to me.  Is there an advantage to doing it this way?  Would it read better as
```rust
if let Ok(file) = fileresult {
```

Reply:
I'm no rust expert, so I can't be sure of performance differences. The only advantage I can see is one less block of indentation, but aside from that I'm not sure